app/views/ops/_rbac_tag_box.html.erb

```diff
@@ -0,0 +1 @@
+<%= settings_rbac_tag_summary()%>
```

Comment:
I think this will work like this 
`<%= settings_rbac_tag_summary%>`

Reply:
made the change. I have habit of calling a function with `()` always